src/components/DropChild.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
```

Comment:
?

Reply:
for these
@typescript-eslint/no-unused-vars
@typescript-eslint/no-use-before-define

command set to error in eslint.config.ts. but v9 have strict rules as compare to v8. your previous v8 script was not working properly. so it showing error for unsed var and no use before define. so we just fix it by adding these lines. to avoid the conflict in code. as we dont know why these variable is there and some are recursive function. thats why.